### DISCUSSION TITLE
Sort by similarity / More Info on hover / Face recognition "Ignore"

### DISCUSSION THREAD

**Discussion by MichaelLurie** [photoprism/photoprism#3074] [Ideas]
I really miss this feature for "cleaning" as I have a lot of duplicates. You can also improve duplicate detection because they were not recognised on upload. 

Would be nice to have some more info showing on hover like date taken or else. Could be configurable on what to show on hover

I have some faces detected by the AI, but I not want to add them. "Ignore" would be nice to have

Thanks and keep going
Michael

**Comment by MichaelLurie** [comment]
Forgot: Archive should also be possible if the pic is in a album. Now I have to remove them first from album and then archive from "unsorted". Kind of complicated. You can warn the user if the pic is attributed to several different albums.

**Comment by graciousgrey** [comment]
Thank you very much for your feedback!

> I have some faces detected by the AI, but I not want to add them. "Ignore" would be nice to have

What should be the difference between "Ignore" and  and [removing faces](https://docs.photoprism.app/user-guide/organize/people/#remove-faces) ? 

>Forgot: Archive should also be possible if the pic is in a album. Now I have to remove them first from album and then archive from "unsorted". Kind of complicated. You can warn the user if the pic is attributed to several different albums.

This is due to the lack of space for more actions in the context menu. You don't need to remove photos from albums to be able to archive them. You can simply select photos in one or multiple albums, then go to "Search" and open the context menu from there. Your selection stays when switching between different sections.

> I really miss this feature for "cleaning" as I have a lot of duplicates. You can also improve duplicate detection because they were not recognised on upload.

At the moment PhotoPrism detects [exact duplicates](https://docs.photoprism.app/user-guide/library/duplicates/) by their SHA1 checksums and sizes. 

Improvements are planned. Related issues:  #1308 #1182 #28
